src/it/analyzer-apiregions-errors-detected/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+    agreements. See the NOTICE file distributed with this work for additional information
+    regarding copyright ownership. The ASF licenses this file to you under the Apache License,
+    Version 2.0 (the "License"); you may not use this file except in compliance with the
+    License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the
+    License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+    either express or implied. See the License for the specific language governing permissions
+    and limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>org.apache.sling</groupId>
+  <artifactId>slingfeature-maven-plugin-test</artifactId>
+  <packaging>jar</packaging>
+  <version>1.0.0-SNAPSHOT</version>
+
+  <name>Apache Sling Features Maven plugin test</name>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>@project.groupId@</groupId>
+        <artifactId>@project.artifactId@</artifactId>
+        <version>@project.version@</version>
+        <extensions>true</extensions>
+        <executions>
+          <execution>
+            <id>analyze</id>
+            <phase>validate</phase>
+            <goals>
+              <goal>analyse-features</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <scans>
+            <scan>
+              <includes>**/*.json</includes>
+              <includeTasks>api-regions-dependencies</includeTasks>
+              <contextConfiguration>
+                <exporting-apis>my-global</exporting-apis>
+                <hiding-apis>my-deprecated</hiding-apis>
```

Comment:
I would prefix these with something common so that it's clear that they belong to the same task, something like:
`
<taskConfiguration>
  <api-regions-exporting>my-global</api-regions-exporting>
  <api-regions-hiding>my-deprecated</api-regions-hiding>
</contextConfiguration>
`

I guess this is along the same lines as what @cziegeler has commented on https://github.com/apache/sling-org-apache-sling-feature-analyser/pull/7#issuecomment-436665548